newsroom/push/publishing.py

```diff
@@ -298,3 +299,27 @@ async def publish_planning_into_event(self, planning: dict[str, Any]) -> Optiona
             # Replace the original document
             await service.update(agenda["_id"], agenda)
             return agenda["_id"]
+
+    async def publish_planning_related_events(self, planning: dict[str, Any], related_event: dict[str, Any]):
+        if not related_event.get("link_type") == "secondary":
+            return None
+
+        service = AgendaItemService()
+        event_id = related_event.get("_id")
+
+        event = await service.find_by_id(event_id)
+        updates: dict[str, Any] = {}
+
+        if event.item_type == AgendaItemType.EVENT and planning and planning.get("guid") not in event.planning_ids:
+            updates["planning_ids"] = event.planning_ids + [planning["guid"]]
```

Comment:
This line will also need to be added in the `publish_planning_into_event` function, so an Event with primary link also get's it's `planning_ids` populated